app/mailers/app_mailer.rb

```diff
@@ -0,0 +1,16 @@
+class AppMailer < ActionMailer::Base
+  def send_welcome_email(user)
+    @user = user
+    mail to: user.email, from: "info@myflix.com", subject: "Welcome to Myflix!"
```

Comment:
This could be pre-filtered. When you are in staging and you want to test this out - to have 1 ..lets say admin address to send all emails to.